tests/test_node_provider_db.py

```diff
@@ -139,6 +139,14 @@ def test_channel_lookup_crud():
     assert ('test-dummy-principal-2', 'dummy-slack-channel-2', 'dummy-telegram-chat-2', 'dummy-telegram-channel-2') in channels
     assert ('test-dummy-principal-1', 'dummy-slack-channel-3', 'dummy-telegram-chat-3', 'dummy-telegram-channel-3') in channels
 
+    channels = node_provider_db.get_channels_as_dict()
+    assert channels['test-dummy-principal-1'] == \
+        {'node_provider_id': 'test-dummy-principal-1', 'slack_channel_name': 'dummy-slack-channel-1', 
+         'telegram_chat_id': 'dummy-telegram-chat-1', 'telegram_channel_id': 'dummy-telegram-channel-1'}
+    assert channels['test-dummy-principal-2'] == \
+        {'node_provider_id': 'test-dummy-principal-2', 'slack_channel_name': 'dummy-slack-channel-2', 
+         'telegram_chat_id': 'dummy-telegram-chat-2', 'telegram_channel_id': 'dummy-telegram-channel-2'}
+
```

Comment:
If you change `get_channels_as_dict` to build the list by the last entry (which I think is a good idea), then you will need to update these tests.